### PR TITLE
Added global css override key to UserMenu component

### PR DESCRIPTION
**Pull request by WiXSL**

### CODE REVIEW

File: packages/ra-ui-materialui/src/layout/UserMenu.tsx [hunk 1]
```diff
@@ -31,7 +34,7 @@ const UserMenu = props => {
     const handleClose = () => setAnchorEl(null);
 
     return (
-        <div className={classes.user}>
```
Comment (fzaninotto) [inline packages/ra-ui-materialui/src/layout/UserMenu.tsx:34]: this is a breaking change. Can you please revert it?

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: packages/ra-ui-materialui/src/layout/UserMenu.tsx (hunk 1 above)
Comment (WiXSL) [reply packages/ra-ui-materialui/src/layout/UserMenu.tsx:34]: Done!

**Review by fzaninotto — APPROVED** [review]

**Comment by djhi** [comment]
I think this should target the `master` branch

**Comment by fzaninotto** [comment]
But the buttons doc is only in next... So let's keep it like that.

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**